src/player.h

```diff
@@ -1191,7 +1182,8 @@ class Player final : public Creature, public Cylinder
 	std::map<uint32_t, DepotChest*> depotChests;
 	std::map<uint32_t, int32_t> storageMap;
 
-	std::vector<OutfitEntry> outfits;
+	std::map<uint16_t, uint8_t> outfits;
```

Comment:
Any specific reason to move from a vector? It could be a vector of `pair<u16, u8>` if you wanted to drop the struct

Reply:
lookup and erase is easier in this case but I'm fine with either way

Reply:
I agree with @EvilHero90, now with c++20 we can use the `contains` method, and everything is more fancy